packages/docs/developer-resources/integrations/checklist.md

```diff
@@ -0,0 +1,64 @@
+
+
+## Custodian/Exchange
+
+Please read more under [Custody](./custody.md), but here is a shortened version:
+
+### Detect Transfers
+cUSD is a token that can be accessed via the ERC20 interface, however the native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to `ETH` on Ethereum.
+
+### Proof of Stake
+Users may want to participate in Celo's Proof of Stake system to help secure the network and earn rewards.
+
+### Authorized Signers
+Celo's core smart contracts use Celo's `Accounts` abstraction to allow balance-moving keys to be held in cold storage, while other keys can be authorized to vote and held in warm storage or even with the user.
+
+### Release Gold
+There is an audited `ReleaseGold` smart contract which allows for the release of `CELO` over a set schedule through which `CELO` might be distributed to a user.
+
+## Wallets
+As wallets, we are loosely defining any application that is meant as key store to allow users to interact and transfer value on the Celo platform.
+
+### Key Derivation
+Celo wallets should follow the [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) for deriving private keys from [BIP39 mnemonics](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki). Celo's key derivation path is at `m/44'/52752'/0'/0`. The first key typically is the `account key` that wallets should register themselves with and accept balance transfers on. The second key can be derived to be an account's `dataEncryptionKey` to allow other users on Celo to encrypt information to.
+
+### Identity Protocol
+Celo has a [lightweight identity protocol](/celo-codebase/protocol/identity) that allows users to address each other via their phone number instead of addressses that Celo wallets should implement. Since user privacy is important, Celo wallets should leverage the built-in [Phone Number Privacy protocol](/celo-codebase/protocol/identity/phone-number-privacy) to protect against large-scale harvesting of user phone numbers.
+
+### Wallet Address
+When transferring assets to an account, wallets should check the receiving account's `walletAddress` at which they want to receive funds at. Use cases might be smart contract accounts that want different recovery characteristics, but receive funds at a different address. Also, `walletAddresses` of `0x0` should indicate that the account requires a different mechanism to acquire the `walletAddress`.
+
+### Transaction metadata
+cUSD (aka StableToken) adds an additional method to the ERC20 interface called `transferWithComment` which allows senders to specify an additional comment that Celo wallets should support. Additionally, comments should be encrypted to the `dataEncryptionKey` when applicable.
```

Comment:
```suggestion
The `StableToken` contract that implements cUSD adds an additional method to the ERC20 interface called `transferWithComment` which allows senders to specify an additional comment that Celo wallets should support. Additionally, comments should be encrypted to the `dataEncryptionKey` when applicable.
```

Reply:
Is there a draft standard for tx metadata we can point to yet?

Reply:
Not yet. I hope it would change with CIP8 but since that's not operational yet, I keep it more open ended here